SECP-service/src/main/java/com/visucius/secp/SECPService.java

```diff
@@ -1,26 +1,29 @@
 package com.visucius.secp;
 
```

Comment:
I have a feeling this file is going to be very big in the future. We might want to consider seperating the different services and having a main controller that just starts them. But I'm fine with doing it this way (as long as its all commented)

Reply:
yes you are right. Once it gets big and hard to maintain. We'll split it :)

Reply:
Sounds good